core/rfb.js

```diff
@@ -245,10 +245,10 @@
     });
 
     this._init_vars();
+    this._cleanupSocket();
```

Comment:
We shouldn't have an open socket, so there should be no need to close it here (see comment below).  The other related stuff (clearing the screen, etc) should probably be in it's own function (at the very least, the name of `_cleanupSocket` is confusing here, IMO).